source/documentation/airflow.md

```diff
@@ -15,66 +19,66 @@ Airflow can be used to:
 
-## Set up an Airflow pipeline
+## Set up an Airflow DAG
 
-To set up an Airflow pipeline, you should:
+To set up an Airflow DAG, you should:
 
 1. Create a new repository from the [Airflow template](https://github.com/moj-analytical-services/template-airflow-python).
 2. Create scripts for the tasks you want to run.
-3. Update configuration files.
+3. Update the Dockerfile and other configuration files.
 4. Push your changes to GitHub.
 5. Create a pull request.
-6. Test the pipeline in your Airflow sandbox.
-7. Create a new release.
+6. Test the DAG in your Airflow sandbox.
+7. Create a new release (and check concourse has built it).
 8. Clone the [`airflow-dags`](https://github.com/moj-analytical-services/airflow-dags) repository from GitHub and create a new branch.
 9. Create a DAG script.
 10. Push your changes to GitHub.
 11. Create a pull request and request review from the Data Engineering team.
 12. Merge the pull request into the master branch.
 
-### Create a new repository from the Airflow template
+### (1) Create a new repository from the Airflow template
 
 To create a new repository from the Airflow template:
 
-1. Go to the [`template-airflow-python`](https://github.com/moj-analytical-services/template-airflow-python) repository.
-2. Select **Use this template**.
-3. Fill in the form:
+- Go to the [`template-airflow-python`](https://github.com/moj-analytical-services/template-airflow-python) repository.
+- Select **Use this template**.
+- Fill in the form:
    - Owner: `moj-analytical-services`
-   - Name: The name of your pipeline prefixied with `airflow-`, for example, `airflow-my-pipeline`
+   - Name: The name of your repo prefixied with `airflow-`, for example, `airflow-my-repo`
    - Privacy: Internal (refer to the [public, internal and private repositories](github.html#public-internal-and-private-repositories) section)
-4. Select **Create repository from template**.
+- Select **Create repository from template**.
 
 This copies the entire contents of the Airflow template to a new repository.
 
-### Clone the repository
+#### **Clone the repository**
 
 To clone the repository:
 
-1.  Navigate to the repository on GitHub.
-2.  Select **Clone or download**.
-3.  Ensure that the dialogue says 'Clone with SSH'. If the dialogue says 'Clone with HTTPS' select **Use SSH**.
-4.  Copy the SSH URL. This should start with `git@`.
-5.  In RStudio, select **File** > **New project...** > **Version control** > **Git**.
-6.  Paste the SSH URL in the **Repository URL** field.
-7.  Select **Create Project**.
+-  Navigate to the repository on GitHub.
+-  Select **Clone or download**.
+-  Ensure that the dialogue says 'Clone with SSH'. If the dialogue says 'Clone with HTTPS' select **Use SSH**.
+-  Copy the SSH URL. This should start with `git@`.
+-  In RStudio, select **File** > **New project...** > **Version control** > **Git**.
+-  Paste the SSH URL in the **Repository URL** field.
+-  Select **Create Project**.
```

Comment:
Because it sits within a broader enumeration - I noticed that there was a subheading for each of the previous enumerated points so I made that explict and got rid of these. I don't think it loses anything by not being numbered, given they aren't referenced anywhere.